homeassistant/components/alarm_control_panel/ifttt.py

```diff
@@ -30,14 +30,25 @@
 DATA_IFTTT_ALARM = 'ifttt_alarm'
 DEFAULT_NAME = "Home"
 
-EVENT_ALARM_ARM_AWAY = "alarm_arm_away"
-EVENT_ALARM_ARM_HOME = "alarm_arm_home"
-EVENT_ALARM_ARM_NIGHT = "alarm_arm_night"
-EVENT_ALARM_DISARM = "alarm_disarm"
+CONF_EVENT_AWAY = "event_arm_away"
+CONF_EVENT_HOME = "event_arm_home"
+CONF_EVENT_NIGHT = "event_arm_night"
+CONF_EVENT_DISARM = "event_disarm"
+CONF_AWAIT_CALLBACK = "await_callback"
+
+DEFAULT_EVENT_AWAY = "alarm_arm_away"
+DEFAULT_EVENT_HOME = "alarm_arm_home"
+DEFAULT_EVENT_NIGHT = "alarm_arm_night"
+DEFAULT_EVENT_DISARM = "alarm_disarm"
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
     vol.Optional(CONF_CODE): cv.string,
+    vol.Optional(CONF_EVENT_AWAY, default=DEFAULT_EVENT_AWAY): cv.string,
+    vol.Optional(CONF_EVENT_HOME, default=DEFAULT_EVENT_HOME): cv.string,
+    vol.Optional(CONF_EVENT_NIGHT, default=DEFAULT_EVENT_NIGHT): cv.string,
+    vol.Optional(CONF_EVENT_DISARM, default=DEFAULT_EVENT_DISARM): cv.string,
+    vol.Optional(CONF_AWAIT_CALLBACK, default=True): cv.boolean,
```

Comment:
Please name this CONF_OPTIMISTIC which is what we use throughout Home Assistant.

Reply:
Ok, I changed it and updated the documentation as well